src/test/java/org/jenkinsci/plugins/workflow/pipelinegraphanalysis/StatusAndTimingTest.java

```diff
@@ -721,7 +736,7 @@ public void queuedAndParallel() throws Exception {
         SemaphoreStep.waitForStart("wait-a/1", b1);
 
 
-        // Now get the end nodes as of the entry of the semaphore on the a branch...
+        // Now get the end nodes as of the entry of the semaphore on the branch...
```

Comment:
I think you misinterpreted this as a grammar mistake.

```suggestion
        // Now get the end nodes as of the entry of the semaphore on the `a` branch...
```